cypress/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "../configs/tsconfig.composite.json",
+    "compilerOptions": {
```

Comment:
I was thinking about using `declarationMap` and `outDir` here, to avoid keeping TS compilation output in the same directories as the source file, but that would likely require an additional `package.json`, and IMO that's too much overhead ATM.

I'll leave this comment here to let you know that I considered this option :smile: and notify you, just in case you did not think of that :slightly_smiling_face: You can resolve it if you agree that it's too much overhead

Reply:
Actually I thought about the same, but I think as this package grows at some point in the future its structure will need to be refactored anyway, possibly by splitting it into smaller packages. But since it's still relatively small I'm not sure how exactly it should look, so for now I'd leave it as it is.

Reply:
Good points, guys. I'm taking that as an early call for splitting single `cloudify-ui-common` package into few packages. Makes sense. Will add it to UI Technical Debt epic.